server/publish.js

```diff
@@ -11,20 +10,8 @@ Meteor.publish('jobs', function() {
   }
 });
 
-Meteor.publish('students', function() {
-  return Roles.getUsersInRole('student', 'user-type', {fields: {'profile.cpf': 0, services: 0}});
-});
-
-Meteor.publish('companies', function() {
-  return Roles.getUsersInRole('company', 'user-type', {fields: {'profile.cnpj': 0, services: 0}});
-});
-
-Meteor.publish('user', function(params) {
-  return Meteor.users.find({_id: params.id}, {fields: {'profile.cnpj': 0, 'profile.cpf': 0, services: 0}});
-});
-
-Meteor.publish('company', function(id) {
-  return Meteor.users.find({_id: id}, {fields: {'profile.cnpj': 0, services: 0}});
+Meteor.publish('users', function() {
+  return Meteor.users.find({}, {fields: {'profile.cpf': 0, 'profile.cnpj': 0, services: 0}});
```

Comment:
I'm publishing all users now. The user will have access to all users with no exception. I was being very careful to publish only the data he needs because of performance, but it was being a pain in the ass to make this, and I don't believe the results would be very significant, so I decided to publish all users to save code and time.